operator/pom.xml

```diff
@@ -443,5 +443,11 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>org.junit.platform</groupId>
+        <artifactId>junit-platform-suite-engine</artifactId>
```

Comment:
what is this new dependency used for?

Reply:
Ah, I had added that temporarily to try to find inefficiencies in the tests. I will remove it.